gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/replication/ReplicationConfiguration.java

```diff
@@ -71,6 +71,8 @@
   public static final String REPLICATION_DATA_CATETORY_TYPE = "replicationDataCategoryType";
   public static final String REPLICATION_DATA_FINITE_INSTANCE = "replicationDataFiniteInstance";
 
+  public static final String COPY_SCHEMA_CHECK_ENABLED = "gobblin.selected.schemaCheck.enabled";
```

Comment:
Why the constant name and value doesn't match? gobblin.selected.schemaCheck.enabled => gobblin.copy.schemaCheck.enabled ?